build.gradle

```diff
@@ -89,6 +89,7 @@ subprojects {
         systemProperty 'io.netty.allocator.smallCacheSize', '0'
         systemProperty 'io.netty.allocator.normalCacheSize', '0'
         systemProperty 'io.netty.allocator.maxCachedBufferCapacity', '0'
+        systemProperty 'io.netty.leakDetection.acquireAndReleaseOnly', 'true'
```

Comment:
what does this setting do?

Reply:
Netty's leak detector will add a record to the stack whenever there is a read, write, create and release operation, this config will disable adding record when read and write, only when create and release.